jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -54,26 +54,53 @@ define([
     PolymerSupport.init();
 
     // dashboard-actions depends on requirejs text plugin
-    require(['./dashboard-actions'], function(DashboardActions) {
+    require(['./dashboard-actions', './dashboard-metadata'], function(DashboardActions, Metadata) {
```

Comment:
My reasoning was that the distinction of 'grid' and 'report' layout would happen at a higher level than `dashboard.js`, which would be a Gridstack layout that can take parameters. The `dashboard-view.js` would modify the parameters to switch layouts between grid and report. We could just as easily bake these layouts into the `dashboard.js` module, but if other non-Gridstack layouts are added in the future they will have to be managed in other placed. I decided it would make the most sense if `dashboard-view.js` managed the layout by instantiating the proper layout modue with the proper parameters for the selected layout.

It may be good to do some module renaming to more accurately specify their roles in the dashboard extension:

`dashboard-view.js` --> `dashboard.js`
`dashboard.js` --> `gridstack-dashboard.js`

In the future there could be additional layout modules such as `slide-dashboard.js` or `multipage-dashboard.js`.